packages/browser/src/core/arguments-resolver/index.ts

```diff
@@ -38,7 +38,7 @@ export function resolveArguments(
     : {}
 
   let opts: Options = {}
-  if (isPlainObject(properties) && !isFunction(options)) {
+  if (!isFunction(options)) {
```

Comment:
Originally had this as `isPlainObject(data)` but `data` will always be an object at this point so it felt unnecessary here.